app/pdfs/distribution_pdf.rb

```diff
@@ -175,38 +175,35 @@ def request_data
       "Value/item",
       "In-Kind Value Received",
       "Packages"]]
-    inventory = View::Inventory.new(@distribution.organization_id)
```

Comment:
Interestingly, this appears to already have not been used. This time rubocop noticed and had me remove it.